src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -55,9 +56,43 @@
   <UsingTask AssemblyFile="$(XPlatSourceBuildTasksAssembly)" TaskName="WritePackageUsageData" />
   <UsingTask AssemblyFile="$(XPlatSourceBuildTasksAssembly)" TaskName="WriteUsageReports" />
 
-  <!-- Update NuGet feeds in the repo -->
+  <Target Name="CopyNuGetConfig"
+          BeforeTargets="
+            UpdateNuGetConfig;
+            Build"
+          Condition="'$(NuGetConfigFile)' != '' or '@(NuGetConfigFiles)' != ''">
+
+    <!-- Update the detected or manually specified NuGetConfigFile, but also allow multiple. -->
+    <ItemGroup>
+      <NuGetConfigFiles Include="$(NuGetConfigFile)" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <ModifiedNugetConfigFilesIntermediateRoot>$(IntermediateOutputPath)nuget.config/</ModifiedNugetConfigFilesIntermediateRoot>
```

Comment:
This should probably be `BaseIntermediateOutputPath` instead. We don't use IntermediateOutputPath here anywhere.

Reply:
Yes, you are correct - will fix.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/55d2441f88af313329b757886fc4d72cf7c177c8